app/classes/Transvision/API.php

```diff
@@ -299,6 +299,11 @@ private function isValidServiceCall($service)
                     No user-defined variables = nothing to check
                 */
                 break;
+            case 'transliterate':
+                if (! $this->verifyEnoughParameters(3)) {
```

Comment:
api/v1/transliterate/serbian/ throws an error "Undefined offset: 3 in /home/theo/moz/transvision/app/models/api/transliterate.php on line 28" We should ask for 4 parameters here